.eslintignore

```diff
@@ -1,9 +1,7 @@
-node_modules
 .loom
-.vscode-test
+.next
+build
+build-internal
+data
```

Comment:
`/stylelint-polaris/data`

Reply:
Good call, will update to put these generated data items in a tmp folder instead. I feel that is a more standard practice and naming convention

Reply:
tmp-tokens even might be a good call to clarify they will be removed with tokens